### PR TITLE
HOUSNAV-191: Results Page Updates

### PR DESCRIPTION
**Pull request by nicholaslyon-dt**
[HOUSNAV-191](https://hous-bssb.atlassian.net/browse/HOUSNAV-191)

## Overview & Purpose
Update the results page (and PDF output) to match the design and be able to handle content for more than 1 walkthrough

## Summary of Changes
- Updated results page to match design and handle more than 1 walkthrough
- Added related building types
- Updated PDF content component to handle more than 1 walkthrough and refactored for maintainability and readability

## Testing
- Complete walkthroughs of with varying building types and answers
- Make sure results page shows correct answers and amounts of answers
- Make sure PDF includes all appropriate questions, answers, and building code content

## Checklist
- [x] I have written or updated vitests for this work
- [x] I have reviewed the [accessibility guidelines](https://digital.gov.bc.ca/wcag/home/) relevant to this work
- [x] I have reviewed this work with at least one screen reader (when applicable)
- [x] I have added/updated any related documentation

### CODE REVIEW

File: packages/data/src/useWalkthroughsTestData.ts [hunk 1]
```diff
@@ -100,7 +100,7 @@ export function getNumberFloatQuestion() {
   };
 }
 
-export function getFirstResult() {
+export function getResultData() {
   // find the first result
```
Comment (brianhoover-willowtree) [inline packages/data/src/useWalkthroughsTestData.ts:104]: If I'm seeing this right, it's still returning the first item in the array, why did we rename it?

Comment (nicholaslyon-dt) [reply packages/data/src/useWalkthroughsTestData.ts:104]: Just to match the naming paradigms in the rest of the file. I'm not sure why I named it that way to begin with.

**Review by jbook24 — APPROVED** [review]

**Review by brianhoover-willowtree — APPROVED** [review]
Looks great, being on the call while these changes were introduced helped considerably!

File: apps/web/cypress/e2e/single-dwelling/single-dwelling-9.9.9-results.cy.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  it("should display correct ui elements", () => {
+    cy.getByTestID(GET_TESTID_RESULT_ITEM(EnumWalkthroughIds._9_9_9)).should(
+      "be.visible",
+    );
+    cy.getByTestID(
+      GET_TESTID_RESULT_RELATED_ITEM(EnumBuildingTypes.MULTI_DWELLING),
+    ).scrollIntoView();
+    cy.getByTestID(
+      GET_TESTID_RESULT_ITEM(EnumBuildingTypes.MULTI_DWELLING),
```
Comment (nicholaslyon-dt) [inline apps/web/cypress/e2e/single-dwelling/single-dwelling-9.9.9-results.cy.ts]: @stechkev This needs to be `GET_TESTID_RESULT_RELATED_ITEM`. Copy paste error.

Comment (stechkev) [reply apps/web/cypress/e2e/single-dwelling/single-dwelling-9.9.9-results.cy.ts]:
```suggestion
      GET_TESTID_RESULT_RELATED_ITEM(EnumBuildingTypes.MULTI_DWELLING),
```

**Pull request merged**